pages/docs/environments/local-development-docker-compose.mdx

```diff
@@ -21,7 +21,7 @@ docker compose up
 
 3) You can now access the application at `http://localhost:3000`.
 
-4) If you making any changes to the dependencies, it's recommended to rebuild the images with the following command. 
+4) If you are making any changes to the dependencies, it's recommended to rebuild the images with the following command. 
```

Comment:
I think it would be better as:
```suggestion
4) If you make any changes to the dependencies, it's recommended to rebuild the images with the following command. 
```
What do you think!? 🤔

Reply:
Yeah sure!